syntax = "proto3";

package mapreduce;


service Reducer {
    rpc Reduce(ReduceTask) returns (ReduceResponse) {}
}

message ReduceTask {
    repeated string partition_paths = 1;
    string output_path = 2;
}

message ReduceResponse {}
